codec-http2/src/test/java/io/netty/handler/codec/http2/Http2MultiplexCodecTest.java

```diff
@@ -211,11 +226,30 @@ public void initChannel(Channel channel) {
         }
     }
 
+    private static ByteBuf bb(String s) {
+        ByteBuf buf = Unpooled.buffer(s.length() * 4);
+        ByteBufUtil.writeUtf8(buf, s);
+        return buf;
+    }
+
     static class LastInboundHandler extends ChannelDuplexHandler {
         private final Queue<Object> inboundMessages = new ArrayDeque<Object>();
         private final Queue<Object> userEvents = new ArrayDeque<Object>();
         private Throwable lastException;
         private ChannelHandlerContext ctx;
+        private boolean channelActive;
+
+        @Override
+        public void channelActive(ChannelHandlerContext ctx) throws Exception {
+            channelActive = true;
+            super.channelActive(ctx);
+        }
+
+        @Override
+        public void channelInactive(ChannelHandlerContext ctx) throws Exception {
+            channelActive = false;
+            super.channelInactive(ctx);
+        }
 
```

Comment:
line 297 ... `writeOutbound` method doesn't appear to be used ... can you remove it?